crates/tools/src/main.rs

```diff
@@ -253,6 +298,28 @@ fn main() {
                 std::process::exit(-1);
             };
         }
+        ("prepare-pk", Some(m)) => {
```

Comment:
This command does a lot of things beyond prepare pks, I think we should rename prepare-pk to setup-nodes